vintf/android.hardware.secure_element.xml

```diff
@@ -0,0 +1,11 @@
+<manifest version="1.0" type="device">
+    <hal format="hidl">
+        <name>ISecureElement</name>
+        <version>1.1</version>
+        <transport>hwbinder</transport>
+        <interface>
+            <name>ISecureElement</name>
+            <instance>eSE1</instance>
```

Comment:
I've seen occurences of a `SIM1` and `SIM2` instance here. We should check what's possible to spawn (on supported platforms, Kumano and Seine, and update the regex).